--- a/modules/luci-mod-system/htdocs/luci-static/resources/view/system/password.js
+++ b/modules/luci-mod-system/htdocs/luci-static/resources/view/system/password.js
@@ -90,7 +90,9 @@
 				known_unix_users[parts[0]] = true;
 		}
 
-		m = new form.JSONMap(formData, _('Password'), _('Changes the password for accessing the device as (rpcd) user'));
+		m = new form.JSONMap(formData, _('Password'),
+			_('Changes the password for accessing the device as (rpcd) user') + '<br />' +
+			_('Firmware community QQ group: 1061612207'));
 		m.readonly = !L.hasViewPermission();
 
 		s = m.section(form.NamedSection, 'data', 'data');
