--- a/package/network/services/uhttpd/files/uhttpd.config
+++ b/package/network/services/uhttpd/files/uhttpd.config
@@ -65,6 +65,9 @@ config uhttpd main
 	# matches have precedence over the CGI prefix.
 #	list ucode_prefix		"/ucode/example=/usr/share/example.uc"
 
+	# List of prefix->proxy handler mappings.
+#	list proxy_prefix		"/proxy/example=http://127.0.0.1:8080"
+
 	# Specify the ubus-rpc prefix and socket path.
 #	option ubus_prefix	/ubus
 #	option ubus_socket	/var/run/ubus/ubus.sock
--- a/package/network/services/uhttpd/files/uhttpd.init
+++ b/package/network/services/uhttpd/files/uhttpd.init
@@ -105,6 +105,18 @@ append_ucode_prefix() {
 	fi
 }
 
+append_proxy_prefix() {
+	local v="$1"
+	local prefix="${v%%=*}"
+	local handler="${v#*=}"
+
+	if [ "$prefix" != "$handler" ] && [ -n "$prefix" ] && [ -n "$handler" ]; then
+		procd_append_param command -F "$prefix=$handler"
+	else
+		echo "Skipping invalid proxy prefix \"$v\"" >&2
+	fi
+}
+
 start_instance()
 {
 	UHTTPD_CERT=""
@@ -159,6 +171,7 @@ start_instance()
 	[ -f /usr/lib/uhttpd_ucode.so ] && {
 		config_list_foreach "$cfg" ucode_prefix append_ucode_prefix
 	}
+	config_list_foreach "$cfg" proxy_prefix append_proxy_prefix
 	append_arg "$cfg" script_timeout "-t"
 	append_arg "$cfg" network_timeout "-T"
 	append_arg "$cfg" http_keepalive "-k"
