diff --git a/luci/luci-app-istorex/luasrc/controller/istorex.lua b/luci/luci-app-istorex/luasrc/controller/istorex.lua
index 700f9a6..6a23cd5 100644
--- a/luci/luci-app-istorex/luasrc/controller/istorex.lua
+++ b/luci/luci-app-istorex/luasrc/controller/istorex.lua
@@ -2,13 +2,9 @@
 module("luci.controller.istorex", package.seeall)
 
 function index()
-    if luci.sys.call("pgrep quickstart >/dev/null") == 0 then
-        entry({"admin", "istorex"}, call("istorex_template")).leaf = true
-        if nixio.fs.access("/usr/lib/lua/luci/view/istorex/main_dev.htm") then
-            entry({"admin", "istorex_dev"}, call("istorex_template_dev")).leaf = true
-        end
-    else
-        entry({"admin", "istorex"}, call("redirect_fallback")).leaf = true
-    end
+    entry({"admin", "istorex"}, call("istorex_template")).leaf = true
+    if nixio.fs.access("/usr/lib/lua/luci/view/istorex/main_dev.htm") then
+        entry({"admin", "istorex_dev"}, call("istorex_template_dev")).leaf = true
+    end
     entry({"admin", "istorex_api","status"}, call("istorex_api_status")).dependent = false
     entry({"admin", "istorex_api","update"}, call("istorex_api_update")).dependent = false
@@ -66,10 +62,6 @@ function get_dev_params()
     return data
 end
 
-function redirect_fallback()
-    luci.http.redirect(luci.dispatcher.build_url("admin","status"))
-end
-
 function istorex_template()
     luci.template.render("istorex/main", get_params())
 end
diff --git a/luci/luci-app-quickstart/luasrc/controller/quickstart.lua b/luci/luci-app-quickstart/luasrc/controller/quickstart.lua
index c54c090..d9c8bb5 100644
--- a/luci/luci-app-quickstart/luasrc/controller/quickstart.lua
+++ b/luci/luci-app-quickstart/luasrc/controller/quickstart.lua
@@ -3,28 +3,24 @@ local http = require "luci.http"
 module("luci.controller.quickstart", package.seeall)
 
 function index()
-    if luci.sys.call("pgrep quickstart >/dev/null") == 0 then
-        local uci = require "luci.model.uci".cursor()
-        entry({"admin", "quickstart"}, template("quickstart/home"), _("QuickStart"), 1).leaf = true
-        entry({"admin", "network_guide"}, call("networkguide_index"), _("NetworkGuide"), 2)
-        entry({"admin", "network_guide", "pages"}, call("quickstart_index", {index={"admin", "network_guide", "pages"}})).leaf = true
-        if uci:get("quickstart", "main", "wifi_menu") == "1" then
-            entry({"admin", "quickwifi"}, call("quickwifi_index"), _("Wireless"), 3)
-            entry({"admin", "quickwifi", "pages"}, call("quickstart_index", {index={"admin", "quickwifi", "pages"}})).leaf = true
-        end
-        if nixio.fs.access("/usr/lib/lua/luci/view/quickstart/main_dev.htm") then
-            entry({"admin", "quickstart_dev"}, call("quickstart_dev", {index={"admin", "quickstart_dev"}})).leaf = true
-        end
-        entry({"admin", "nas", "raid"}, call("quickstart_index", {index={"admin", "nas"}}), _("RAID"), 10).leaf = true
-        entry({"admin", "nas", "smart"}, call("quickstart_index", {index={"admin", "nas"}}), _("S.M.A.R.T."), 11).leaf = true
-        entry({"admin", "network", "interfaceconfig"}, call("quickstart_index", {index={"admin", "network"}}), _("NetworkPort"), 11).leaf = true
+    local uci = require "luci.model.uci".cursor()
+    entry({"admin", "quickstart"}, template("quickstart/home"), _("QuickStart"), 1).leaf = true
+    entry({"admin", "network_guide"}, call("networkguide_index"), _("NetworkGuide"), 2)
+    entry({"admin", "network_guide", "pages"}, call("quickstart_index", {index={"admin", "network_guide", "pages"}})).leaf = true
+    if uci:get("quickstart", "main", "wifi_menu") == "1" then
+        entry({"admin", "quickwifi"}, call("quickwifi_index"), _("Wireless"), 3)
+        entry({"admin", "quickwifi", "pages"}, call("quickstart_index", {index={"admin", "quickwifi", "pages"}})).leaf = true
+    end
+    if nixio.fs.access("/usr/lib/lua/luci/view/quickstart/main_dev.htm") then
+        entry({"admin", "quickstart_dev"}, call("quickstart_dev", {index={"admin", "quickstart_dev"}})).leaf = true
+    end
+    entry({"admin", "nas", "raid"}, call("quickstart_index", {index={"admin", "nas"}}), _("RAID"), 10).leaf = true
+    entry({"admin", "nas", "smart"}, call("quickstart_index", {index={"admin", "nas"}}), _("S.M.A.R.T."), 11).leaf = true
+    entry({"admin", "network", "interfaceconfig"}, call("quickstart_index", {index={"admin", "network"}}), _("NetworkPort"), 11).leaf = true
 
-        entry({"admin", "nas", "quickstart"}).dependent = false
-        entry({"admin", "nas", "quickstart", "auto_setup"}, post("auto_setup"))
-        entry({"admin", "nas", "quickstart", "setup_result"}, call("setup_result"))
-    else
-        entry({"admin", "quickstart"}, call("redirect_fallback")).leaf = true
-    end
+    entry({"admin", "nas", "quickstart"}).dependent = false
+    entry({"admin", "nas", "quickstart", "auto_setup"}, post("auto_setup"))
+    entry({"admin", "nas", "quickstart", "setup_result"}, call("setup_result"))
 end
 
 function networkguide_index()
@@ -35,10 +31,6 @@ function quickwifi_index()
     luci.http.redirect(luci.dispatcher.build_url("admin", "quickwifi", "pages", "quickwifi"))
 end
 
-function redirect_fallback()
-    luci.http.redirect(luci.dispatcher.build_url("admin", "status"))
-end
-
 local function vue_lang()
     local i18n = require("luci.i18n")
     local lang = i18n.translate("quickstart_vue_lang")
